popup.html

```diff
@@ -0,0 +1,33 @@
+<style>
+html {
+    background-color: #232628;
+    color: #CCCCCC;
```

Comment:
Is there a way to make the color scheme in here respect your existing color scheme somehow?